src/components/CubeSearchNavBar.js

```diff
@@ -0,0 +1,245 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { Button, Modal, ModalBody, ModalFooter, ModalHeader, Navbar, Input, CardBody } from 'reactstrap';
+
+import TextField from 'components/TextField';
+import NumericField from 'components/NumericField';
+import SelectField from 'components/SelectField';
+
+const AdvancedSearchModal = ({ isOpen, toggle }) => {
+  const [name, setName] = useState('');
+  const [owner, setowner] = useState('');
+  const [decks, setDecks] = useState('');
+  const [cards, setCards] = useState('');
+  const [include, setInclude] = useState('');
+  const [category, setCategory] = useState('');
+
+  const [decksOp, setDecksOp] = useState('=');
+  const [cardsOp, setCardsOp] = useState('=');
+
+  const Categories = [
+    '',
+    'Vintage',
+    'Legacy+',
+    'Legacy',
+    'Modern',
+    'Pioneer',
+    'Standard',
+    'Set',
+    'Powered',
+    'Unpowered',
+    'Pauper',
+    'Peasant',
+    'Budget',
+    'Silver-bordered',
+    'Commander',
+  ];
+  const handleChange = (event) => {
+    const { target } = event;
+    const value = ['checkbox', 'radio'].includes(target.type) ? target.checked : target.value;
+    const key = target.name;
+
+    switch (key) {
+      case 'name':
+        setName(value);
+        break;
+      case 'owner':
+        setowner(value);
+        break;
+      case 'decks':
+        setDecks(value);
+        break;
+      case 'cards':
+        setCards(value);
+        break;
+      case 'decksOp':
+        setDecksOp(value);
+        break;
+      case 'cardsOp':
+        setCardsOp(value);
+        break;
+      case 'category':
+        setCategory(value);
+        break;
+      case 'include':
+        setInclude(value);
+        break;
+      default:
+        break;
+    }
+  };
+
+  const submit = () => {
+    console.log(name, owner, decks, cards);
```

Comment:
is this needed?

Reply:
darn, I'm surprised that wasn't caught by linter though

Reply:
we globally allow `console` since that was the logging system at the start:
https://github.com/dekkerglen/CubeCobra/blob/master/.eslintrc.yml#L18